mailet/icalendar/src/test/java/org/apache/james/transport/mailets/ICALToJsonAttributeTest.java

```diff
@@ -284,8 +278,48 @@ private String toJsonValue(byte[] ics) {
         return new String(BufferRecyclers.getJsonStringEncoder().quoteAsUTF8(new String(ics, StandardCharsets.UTF_8)), StandardCharsets.UTF_8);
     }
 
+
+    @Test
+    void serviceShouldAttachJsonWithTheReplyToAttributeValueWhenPresent() throws Exception {
+        testee.init(FakeMailetConfig.builder().build());
+
+        byte[] ics = ClassLoaderUtils.getSystemResourceAsByteArray("ics/meeting.ics");
+        Calendar calendar = new CalendarBuilder().build(new ByteArrayInputStream(ics));
+        ImmutableMap<String, Calendar> icals = ImmutableMap.of("key", calendar);
+        ImmutableMap<String, byte[]> rawIcals = ImmutableMap.of("key", ics);
+        MailAddress recipient = MailAddressFixture.ANY_AT_JAMES2;
+        MailAddress replyTo = MailAddressFixture.OTHER_AT_JAMES;
+        Mail mail = FakeMail.builder()
+            .name("mail")
+            .sender(SENDER)
+            .recipient(recipient)
+            .attribute(new Attribute(ICALToJsonAttribute.DEFAULT_SOURCE, AttributeValue.ofAny(icals)))
+            .attribute(new Attribute(ICALToJsonAttribute.DEFAULT_RAW_SOURCE, AttributeValue.ofAny(rawIcals)))
+            .mimeMessage(MimeMessageBuilder.mimeMessageBuilder()
+                .addHeader(ICALToJsonAttribute.REPLY_TO_HEADER_NAME, replyTo.asString()))
```

Comment:
We need a wider coverage, header can be messy...

We need support for the following syntaxes:

```
Reply-To: iv@lid@domain.tld // we must ignore it

Reply-To: bob@domain.tld
Reply-To: <bob@domain.tld>

Reply-To: "Bob" <bob@domain.tld>

Reply-To: "Bob"
 <bob@domain.tld>

Reply-To: "QUOTED ENCODING" <bob@domain.tld>
```

Sorry mime address headers are messy...